src/services/userService.js

```diff
@@ -18,6 +18,10 @@ class UserService {
   signUp(user) {
     return api.post(prefix, user);
   }
+
+  getProfile(id) {
+    return api.get(`${prefix}/${id}`);
```

Comment:
What is `prefix`?

Reply:
I defined this constante for each service file. It is at the top of the file:
`const prefix = '/users';`